### ISSUE TITLE
Crash when clicking the 'help you get started' links to open finder

### ISSUE THREAD

**Issue by brian-mann** [cypress-io/cypress#227]
Clicking `example_spec.js` or `cypress/integration` results in an immediate crash.

This is a regression from `0.17.0`.

![screen shot 2016-09-16 at 4 17 06 pm](https://cloud.githubusercontent.com/assets/1268976/18601567/2ee7fbe2-7c29-11e6-9d32-d2a6883dda66.png)

**Comment by jennifer-mann** [comment]
This is a problem in `cypress-app`

**Error emitted on click of link:**
```bash
TypeError: open.opn is not a function
  at EventEmitter.handleEvent (.../cypress-app/lib/electron/handlers/events.coffee:117:12)
  at apply (.../cypress-app/node_modules/lodash/lodash.js:435:17)
  at EventEmitter.wrapper (.../cypress-app/node_modules/lodash/lodash.js:4790:16)
  at emitMany (events.js:127:13)
  at EventEmitter.emit (events.js:201:7)
  at EventEmitter.<anonymous> (.../cypress-app/node_modules/@cypress/core-electron/dist/Cypress/Cypress.app/Contents/Resources/electron.asar/browser/api/web-contents.js:156:13)
  at emitTwo (events.js:106:13)
  at EventEmitter.emit (events.js:191:7)
```

**Comment by brian-mann** [comment]
Fixed in [`0.19.1`](https://github.com/cypress-io/cypress/wiki/Changelog#0191-03092017).
